packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer-selectors.ts

```diff
@@ -53,4 +59,17 @@ export const GeneratedAnswerSelectors: GeneratedAnswerSelector = {
     GeneratedAnswerSelectors.get().find(
       '[data-cy="generated-answer__retry-button"]'
     ),
+  feedbackModal: () => cy.get('lightning-modal'),
+  feedbackOption: (index: number) =>
```

Comment:
Would it make sense to have a `feedbackOption: (text: string)` ? Like a select option by text instead of by index?
Just an idea, you have already coded it without it so feel free to disregard :)